ember.md

```diff
@@ -183,3 +185,23 @@ CSS is permitted (and encouraged) in apps and addons under certain circumstances
 </div>
 
 ```
+<a name="testing"></a>
+## Testing
+
+<a name="testing--test-scripts"></a>
+### 5.1 Test Scripts
+
+> Why? To allow us to establish and hold to a standard of code coverage in all of our apps with meaningful test writing and the ability to gate deployments when those standards are not met.
+
+We use two devDependencies to compile our test scripts.  
+1. `cross-env` - For allowing single commands for multiple test environments
```

Comment:
the repo says this if for windows usage: https://github.com/kategengler/ember-cli-code-coverage/

> If you want your coverage to work on both Unix and Windows, you can do this:
> 
> npm install cross-env --save-dev

Reply:
@ASH-Anthony Is there a specific revision that you have in mind?

Reply:
what do you mean by "multiple test envs?" `cross-env` is needed because we run windows. is that what you're referring to?

Reply:
I see what you are saying. I can update that verbiage.

Reply:
Something like:
`cros-env` - For properly setting the NODE_ENV on Windows test environments

Reply:
i like that

Reply:
You might want to say `For setting environment variables in a cross-platform manner.` Just because this isn't specific to node.

Reply:
@ASH-Bryan The previous statement was more general, though. I think this change specifies why we here at ASH use it. Thoughts?

Reply:
I was just thinking in terms of people understanding that cross-env can be used to set env vars in other contexts - like other scripts they might want to use from package.json.